eqcorrscan/utils/catalog_to_dd.py

```diff
@@ -787,8 +878,16 @@ def write_correlations(catalog, stream_dict, extract_len, pre_pick,
         max_sep=max_sep, min_link=min_link, max_workers=max_workers,
         stream_dict=processed_stream_dict, min_cc=min_cc,
         extract_len=extract_len, pre_pick=pre_pick, shift_len=shift_len,
-        interpolate=interpolate, all_horiz=all_horiz, **kwargs)
-    with open("dt.cc", "w") as f:
+        interpolate=interpolate, all_horiz=all_horiz, 
+        existing_corr_file=existing_corr_file, **kwargs)
+    with open(output_filename, "w") as f:
+        if existing_corr_file is not None:
+            with open(existing_corr_file, "r") as in_f:
+                for line in in_f:
+                    f.write(line)
+        ## TODO: speed up saving existing correlations output (do I need to write out every line individually?)
```

Comment:
If you want to just dump everything you can do something like:
```python
with open(blah) as f:
    f.write(["\n".join(l for l in in_f])
```
assuming that your lines don't already have a newline character at the end of them. If they do then you can just `''.join(in_f)` and write that string.